docs/en_US/ruleset-update-channels.md

```diff
@@ -0,0 +1,177 @@
+3. You would like to implement tracker blocking within HTTPS Everywhere by forwarding a list of hosts to an unroutable address
+
+There may be additional use cases not enumerated here.  For this to be effective, an organization has to publish their own custom update channel.  This involves a few steps:
+
+1. Creating an RSA key and generating a `jwk` object from it
+2. Signing rulesets with this key
+3. Publishing those rulesets somewhere
+4. Getting users to use your update channel
+
+We will go through each of these in sequence, but first, you'll want to consider if you want your signing process airgapped or not.  Airgapped signing has the advantage of making it hard for malware to exfiltrate key material and thus forge a signed ruleset update, but it will also make it slightly more difficult to sign.  If you decide on an airgapped signing process, you may want to copy the script [`utils/sign-rulesets/async-airgap.sh`](https://github.com/EFForg/https-everywhere/blob/master/utils/sign-rulesets/async-airgap.sh) to it *before* cutting off networking for the last time.  You may also want to install the `python-qr` code on this machine to easily copy the RSA public key to your development environment, once generated, as well as `qrencode` and `eog` for ease in the signing process.
+
+### 1. Creating an RSA key and generating a `jwk` object from it
+
+To create an RSA key, issue the following command (either on your development machine or an airgap):
+
+    openssl genrsa -out key.pem 4092
+
+Your RSA keypair will now be stored in the file `key.pem`.  To generate a corresping public key, issue this command:
+
+    openssl rsa -in key.pem -outform PEM -pubout -out public.pem
+
+Your public key is now stored in `public.pem`.  If you are using an airgap, copy this public key (with whatever method is safest in your setup, perhaps with the `qr` command) to your development environment.
+
+At this point, you will need to generate a `jwk` object from the public key.  This can be done with the `pem-jwk` node package.  You'll have to download node.js and npm, or just issue this command in docker:
+
+    sudo docker run -it -v $(pwd):/opt --workdir /opt node bash
+
+And you will be booted into a node environment.  Next, run
+
+    npm install -g pem-jwk
+
+This will install the `pem-jwk` package globally.  You can now run
+
+    cat public.pem | pem-jwk
+
+And you should see a `jwk` object displayed.  Take note of this, you will need it later.
+
+### 2. Signing rulesets with this key
+
+#### Setup
+
+On your development machine, clone or download the HTTPS Everywhere repository.  Since it's quite large, it will suffice to do a shallow clone:
+
+    git clone --depth=1 https://github.com/EFForg/https-everywhere.git
+
+or
+
+    curl -sLO https://github.com/EFForg/https-everywhere/archive/master.zip; unzip master.zip; mv https-everywhere-master https-everywhere
+
+Next,
+
+    cd https-everywhere
+    rm rules/*.xml
```

Comment:
One can save the time `rm` the existing rulesets by pointing `utils/merge-rulesets.py` to the correct directory. 

https://github.com/EFForg/https-everywhere/blob/b87b31b1b1ff98b1902ea48e1d36ed682ef13f15/utils/merge-rulesets.py#L25-L27

Reply:
I want to avoid the possibility that a ruleset channel maintainer might forget this `--source_dir` flag in the future, and I don't see any harm in simply removing the `xml` files in `rules` for anyone maintaining a non-EFF ruleset channel.  I'm inclined to keep it, since adding a `mkdir` and pointing to it using `--source_dir` seems about as complicated as what we currently have here.